omnipaxos/src/sequence_paxos/follower.rs

```diff
@@ -167,26 +162,18 @@ where
                         ],
                         "storage error while trying to write log entries",
                     );
-                    Accepted {
-                        n: accsync.n,
-                        accepted_idx: accepted_idx
-                            .expect("storage error while trying to write log entries"),
-                    }
                 }
             };
-            match accsync.stopsign {
-                Some(ss) => {
-                    if let Some(ss_entry) = self.internal_storage.get_stopsign() {
-                        if !ss_entry.decided(old_decided_idx) {
-                            self.accept_stopsign(ss);
-                        }
-                    } else {
-                        self.accept_stopsign(ss);
-                    }
-                    accepted.accepted_idx = self.internal_storage.get_accepted_idx();
-                }
-                None => self.forward_pending_proposals(),
+            if accsync.stopsign.is_none() {
+                self.forward_pending_proposals();
             }
+            self.internal_storage
+                .set_stopsign(accsync.stopsign)
+                .expect("storage error while trying to write stopsign");
```

Comment:
somewhere in this function, we need to set the decided_idx set by the leader.

Reply:
This happens on line 102